cassandra-analytics-integration-tests/src/test/java/org/apache/cassandra/analytics/BulkWriteUdtTest.java

```diff
@@ -207,13 +210,14 @@ private int populateSetOfUdts()
         // table(id, set<udt(list<>, set<>, map<>)>)
         // insert set of UDTs, and UDT has a list, set and map inside it
         String insertIntoSetOfUdts = "INSERT INTO %s (id, udtset) VALUES (%d, " +
-                "{{f1:['value %d'], f2:{%d}, f3:{%d : 'value %d'}}})";
+                                     "{{f1:['list value %d'], f2:{'set value %d'}, f3:{%d : 'map value %d'}, " +
```

Comment:
@yifan-c this tests set of UDTs of collections, tuple is one of collection. I need to update above comment to table(id, set<udt(list<>, set<>, map<>, tuple<>)>) for clarity